### ISSUE TITLE
H2: Support DSL#execute(String)

### ISSUE THREAD

**Issue by knutwannheden** [jOOQ/jOOQ#8747]
It should be possible to support `DSL#execute(String)` in H2.

**Comment by lukaseder** [comment]
I wonder if this really makes sense at this point. `EXECUTE (IMMEDIATE)` is for running dynamic SQL or dynamic procedural code in procedural languages. This seems quite difficult to achieve in H2 stored procedures. We'd need access to a Java compiler...

**Comment by knutwannheden** [comment]
Right. I thought it was for SQL only, in which case Connection#createStatement() should work.

For procedural code I don't even know what the user should expect on H2...

Should the API have distinct API for the two use cases?

**Comment by lukaseder** [comment]
> Should the API have distinct API for the two use cases?

At this point, I prefer not to overengineer this particular emulation. I still hope H2 eventually implements a minimal procedural language :)

**Comment by katzyn** [comment]
H2 1.4.200 has `EXECUTE IMMEDIATE`:
https://h2database.com/html/commands.html#execute_immediate
(and still doesn't have pure SQL procedures).
